### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by paraita**
Some tests were legitimaly failing. The gruntfile has been updated and fixed so we can now build a proper version of the studio with minified css/js files. It is also possible to run the tests locally without running a real scheduler.

### CODE REVIEW

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

File: test/spec/ImportExport.js [hunk 1]
```diff
@@ -1,131 +1,134 @@
...
-                    }
+define(['proactive/view/xml/JobXmlView', 'proactive/model/Job', 'proactive/view/utils/undo', 'xml2json', 'prettydiff'],
+    function (JobXmlView, Job, undoManager, xml2json, prettydiff) {
+
+        "use strict";
+
+        return describe("Jobs Import / Export", function () {
+
+            var readFromServer = function (fileName) {
+                console.log("Reading " + httpServer + fileName);
```
Comment (sophiesongge) [inline test/spec/ImportExport.js]: It is better to remove the console.log

File: test/spec/ImportExport.js [hunk 2]
```diff
@@ -1,131 +1,134 @@
...
+            var path = '';
+            for (var i = 0; i < pathArr.length - 2; i++) {
+                path += pathArr[i] + '/';
+            }
+            var httpServer = location.protocol + "//" + location.host + path;//"http://localhost:8000/";
+
+            var jobList = $(readFromServer('/test/jobs/'));
+            jobList.find('[href*=".xml"]').each(function (i, jobHtml) {
+                var jobName = $(jobHtml).text()
+                console.log(undoManager)
```
Comment (sophiesongge) [inline test/spec/ImportExport.js]: better to remove the console.log

Comment (sophiesongge) [inline test/spec/ImportExport.js]: better to remove the console.log, and pay attention to the ";"

File: test/spec/ImportExport.js [hunk 3]
```diff
@@ -1,131 +1,134 @@
...
+                        originalJobXml = originalJobXml.replace(/<controlFlow block="start" \/>/g, '<controlFlow block="start"> </controlFlow>')
+
+                        generatedXml = generatedXml.replace(/xmlns:xsi=".*"/, '')
+                        generatedXml = generatedXml.replace(/xmlns=".*"/, '')
+                        generatedXml = generatedXml.replace(/xsi:schemaLocation=".*"/, '')
+                        generatedXml = generatedXml.replace(/\n+/g, ' ')
+                        generatedXml = generatedXml.replace(/\s+/g, ' ')
+                        generatedXml = generatedXml.replace(/></g, '> <')
+
+                        console.log("old: ", originalJobXml)
```
Comment (sophiesongge) [inline test/spec/ImportExport.js]: console.log :)

**Comment by paraita** [comment]
retest this please

**Pull request merged**